src/sema/builtin.rs

```diff
@@ -1305,6 +1305,17 @@ pub fn resolve_method_call(
         .collect();
     let marker = diagnostics.len();
 
+    // check if the arguments are not garbage
+    let mut errors = false;
+
+    for arg in args {
+        errors |= expression(arg, context, ns, symtable, diagnostics, ResolveTo::Unknown).is_err();
+    }
+
+    if errors {
+        return Err(());
+    }
+
```

Comment:
This is exactly the same code from here:
https://github.com/hyperledger-labs/solang/blob/6a2ba29d45b808c6a526d7a9be743e8337f5727c/src/sema/builtin.rs#L881-L891
How about creating a function?

Reply:
I've introduce `fn args_sanity_check()`